cmd/trace-agent/config/config.go

```diff
@@ -193,6 +193,12 @@ func applyDatadogConfig(c *config.AgentConfig) error {
 	if coreconfig.Datadog.IsSet("apm_config.connection_limit") {
 		c.ConnectionLimit = coreconfig.Datadog.GetInt("apm_config.connection_limit")
 	}
+	if coreconfig.Datadog.IsSet("apm_config.peer_service_stats_aggregation") {
+		c.PeerServiceStatsAggregation = coreconfig.Datadog.GetBool("apm_config.peer_service_stats_aggregation")
+	}
+	if coreconfig.Datadog.IsSet("apm_config.compute_stats_by_span_kind") {
+		c.ComputeStatsBySpanKind = coreconfig.Datadog.GetBool("apm_config.compute_stats_by_span_kind")
+	}
```

Comment:
```suggestion
	c.PeerServiceStatsAggregation = coreconfig.Datadog.GetBool("apm_config.peer_service_stats_aggregation")
	c.ComputeStatsBySpanKind = coreconfig.Datadog.GetBool("apm_config.compute_stats_by_span_kind")
```
You already set a default in `pkg/config/apm.go`